bitnami/metrics-server/templates/deployment.yaml

```diff
@@ -49,6 +49,9 @@ spec:
       hostNetwork: true
       {{- end }}
       containers:
+        {{- if .Values.extraContainers }}
+        {{- toYaml .Values.extraContainers | nindent 8 }}
```

Comment:
In other charts we have this:
```yaml
{{- include "common.tplvalues.render" (dict "value" .Values.extraContainers "context" $) | nindent 8 }}
```
Could you change it?

Reply:
@Mauraza Change applied, thanks.